README.md

```diff
@@ -2,8 +2,14 @@
 
 This module implements the [`rdk:component:board` API](https://docs.viam.com/components/board/#api) and [`rdk:component:servo` API](https://docs.viam.com/components/servo/#api)
 
-Two models are provided in this module:
-* `viam:raspberry-pi:rpi` - Configure a Raspberry Pi 4, 3 and Zero 2 W,  board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+The following models are provided in this module:
+* `viam:raspberry-pi:rpi5` - Configure a Raspberry Pi 5 board to access GPIO functionality: input, output, PWM, power, serial interfaces, etc.
+* `viam:raspberry-pi:rpi4` - Configure a Raspberry Pi 4 board
+* `viam:raspberry-pi:rpi3` - Configure a Raspberry Pi 3 board
+* `viam:raspberry-pi:rpi2` - Configure a Raspberry Pi 2 board
+* `viam:raspberry-pi:rpi1` - Configure a Raspberry Pi 1 board
+* `viam:raspberry-pi:rpi0` - Configure a Raspberry Pi 0 board
+* `viam:raspberry-pi:rpi0_2` - Configure a Raspberry Pi 0 2 W board
 * `viam:raspberry-pi:rpi-servo` - Configure a servo controlled by the GPIO pins on the board.
```

Comment:
Will the rpi-servo work with pi5? I think the answer is no so we should specify this in readme

Reply:
i believe it will work? rand and I had actually talked about removing the rpiservo logic to just use the gpio servo logic in rdk, but we never got around to it

Reply:
oh okay I assumed it didn't because it uses pigpio to do PWM which doesn't support the pi5? But i am in support of removing it